Tests/UnitTests/Purchasing/ErrorCodeTests.swift

```diff
@@ -181,3 +181,100 @@ class ErrorCodeTests: TestCase {
     }
 
 }
+
+import StoreKit
+
+class ErrorUtilsTests: TestCase {
+
+    private var originalLogHandler: VerboseLogHandler!
+    private var loggedMessages: [(level: LogLevel, message: String)] = []
+
+    override func setUp() {
+        super.setUp()
+
+        self.originalLogHandler = Logger.logHandler
+        Logger.logHandler = { level, message, _, _, _ in
+            self.loggedMessages.append((level, message))
```

Comment:
I think it might be ok for the tests, but should we use a `weak self` here?  (Been a while since I did some iOS development)

Reply:
The log handler is restored, so the cycle is broken, but it doesn't hurt 👍🏻

Reply:
Well there's no cycle, but we probably don't want to make the log handler have a strong reference to the test instance.

Reply:
Done